[2024-07-10T09:28:51.950+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-10T09:28:51.959+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data manual__2024-07-10T09:28:47.551323+00:00 [queued]>
[2024-07-10T09:28:51.962+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data manual__2024-07-10T09:28:47.551323+00:00 [queued]>
[2024-07-10T09:28:51.962+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-10T09:28:51.966+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): concat_data> on 2024-07-10 09:28:47.551323+00:00
[2024-07-10T09:28:51.969+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3323) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-10T09:28:51.970+0000] {standard_task_runner.py:63} INFO - Started process 3332 to run task
[2024-07-10T09:28:51.970+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'concat_data', 'manual__2024-07-10T09:28:47.551323+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/sql_2.py', '--cfg-path', '/tmp/tmpequ70up3']
[2024-07-10T09:28:51.971+0000] {standard_task_runner.py:91} INFO - Job 401: Subtask concat_data
[2024-07-10T09:28:51.986+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.concat_data manual__2024-07-10T09:28:47.551323+00:00 [running]> on host 10d335c2a089
[2024-07-10T09:28:52.014+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='concat_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-10T09:28:47.551323+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-10T09:28:47.551323+00:00'
[2024-07-10T09:28:52.015+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-10T09:28:52.018+0000] {base.py:84} INFO - Using connection ID 'Postgres_server' for task execution.
[2024-07-10T09:28:52.166+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-10T09:28:52.167+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-10T09:28:52.170+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=concat_data, run_id=manual__2024-07-10T09:28:47.551323+00:00, execution_date=20240710T092847, start_date=20240710T092851, end_date=20240710T092852
[2024-07-10T09:28:52.195+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-10T09:28:52.205+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-10T09:28:52.205+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
